NodeJS/README.md

```diff
@@ -0,0 +1,197 @@
+--network unleash --name postgres postgres
+```
+
+3. Start Unleash via docker (open another terminal):
+
+```sh
+docker run -p 4242:4242 \
+-e DATABASE_HOST=postgres -e DATABASE_NAME=unleash \
+-e DATABASE_USERNAME=unleash_user -e DATABASE_PASSWORD=some_password \
+-e DATABASE_SSL=false \
+--network unleash unleashorg/unleash-server
+```
+
+Now if we open http://localhost:4242 we should get something like this 👇  
+<br/>
+<img src="./static/img/login.png" title="Login page Unleash" />
+
+
+You can login with the default admin user:
+
+- username: `admin`
+- password: `unleash4all`
+
+Once you are in go ahead and create a feature toggle. [Don't know how ?](https://docs.getunleash.io/user_guide/create_feature_toggle)
+
+### Step 2: Create an Express application
+
+Now let's create a simple server using ExpressJS;
+
+1. Create a folder (for example: NodeJS);
+2. Open terminal and navigate to the folder
+3. Run `npm init -y`, this should create a file called _package.json_.
+4. Now create a file called _server.js_
+5. Back to the terminal, we need to install express so run:  
+```sh
+npm install express
+```
+7. Now add the following code to the _server.js_ file:
+
+```js
+const express = require("express");
+const app = express();
+const port = 3030;
+
+app.get("/", (req, res) => {
+  res.status(200).send("Example using Unleash with NodeJS + ExpressJS");
+});
+
+app.listen(port, () => {
+  console.log(`Example app listening at http://localhost:${port}`);
+});
+```
+
+8. Start the server by typing this command in the terminal:
+
+```sh
+node server.js
+```
+
+now when you open http://localhost:3030 in your browser you should get something like this👇  
+
+<br/>
+<img src="./static/img/express-example.png" title="express-example" />
+
+### Step 3: Connect the SDK with our app
+
+Now in order to use the feature we created earlier in _step 1_ with our express app we need to use the NodeJS Client SDK.
+
+1. Install the sdk using npm:
+
+```sh
+npm install unleash-client
+```
+
+2. Next we must initialize the client SDK:
+
+```js
+const { initialize } = require("unleash-client");
+const unleash = initialize({
+  url: "http://localhost:4242/api/",
+  appName: "my-node-name",
+  instanceId: "my-unique-instance-id",
+  customHeaders: {
+    Authorization: "API TOKEN"
+  },
+});
+```
+
+We need to replace the `API TOKEN` with a real one. So let's go and create a _Client API Key_. [Here's how](https://docs.getunleash.io/user_guide/api-token)
+
+Now let's check if the SDK is connected.
+
+first import isEnabled from unleash-client:
+```js
+const { initialize, isEnabled } = require("unleash-client");
+```
+
+Now add the code below:
+
+```js
+setInterval(() => {
+    if (unleash.isEnabled('DemoToggle'))
+    {    console.log('Toggle enabled');  }
+    else {    console.log('Toggle disabled');  }
+    }, 1000);
+```
+**Note ⚠️** : `DemoToggle` should be replaced with your feature toggle name that you created.
```

Comment:
Should we also describe that the fist time you do the isEnabled call the SDK may not have had a chance to synchronize with the Unleash API yet, and that you need to listen for the ready event (or away the promise returned from the initialize method) if you want to be guaranteed that it is fully synchronized? 

The Node SDK will on purpose not block the customers application, but the user can choose to block until the SDK has fully synchronized.